gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -188,22 +187,31 @@ def run_assertion(self):
 class QueryFormatter(object):
   """Formats a query.
 
-  Replaces keyword TABLE_NAME and eventually macros in the query.
+  Replaces keyword TABLE_NAME and macros in the query.
   """
 
+  NUM_ROWS = 'SELECT COUNT(0) AS num_rows FROM {TABLE_NAME}'
```

Comment:
I think it's cleaner to create an enum of these macros. It both provides a natural grouping of these constants and also is easier to loop through them (e.g. adding a new macro is just as easy as adding a new entry in that Enum).
So, it would be something like:
```
import enum
class _QueryMacros(enum.Enum):  # nested under the QueryFormatter class
  NUM_ROWS_QUERY = 'SELECT ...'
  SUM_START_QUERY = 'SELECT ...'

def _replace_macros(query):
  for macro in _QueryMacros:
    if macro.name == query:
      return macro.value
  else:
    return query 
``` 

This also means that you'd no longer need the surrounding `{}` brackets in the test def, which further distinguishes macros from regular variable names.

Reply:
Thanks for such a detailed comment! DONE.

Reply:
I actually got a warning on "for macro in _QueryMacros", says "Expected expected collections.iterable got _QueryMacros instead. Any idea how to fix this? I have found one way which loops through the __members__.items, but I don't think it is as readable as current code.